src/tests/audio/tasks/speaker_verification_test.py

```diff
@@ -0,0 +1,66 @@
+"""Test Module for Audio Processing and Speaker Verification.
+
+This module contains minimal tests to ensure the audio processing and speaker verification functions do not fail.
+
+Tests:
+    - test_resample_iir: Tests the resample_iir function.
+    - test_verify_speaker: Tests the verify_speaker function.
+    - test_verify_speaker_from_files: Tests the verify_speaker_from_files function.
+"""
+
+import os
+
+import pytest
+import torch
+
+from senselab.audio.data_structures.audio import Audio
+from senselab.audio.tasks.speaker_verification.speaker_verification import (
+    _resample_iir,
+    verify_speaker,
+    verify_speaker_from_files,
+)
+
+MONO_AUDIO_PATH = "src/tests/data_for_testing/audio_48khz_mono_16bits.wav"
+
+
+@pytest.fixture
```

Comment:
please, use the global fixture as in the other tests

Reply:
Ohhh, I don't have to import it--couldn't import it, so I just copied it. I see that I can just use it now

Reply:
Done.